### PR TITLE
feat: auto hide topbar and filters sticky

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

**Comment by stepan662** [comment]
Improve scrolling detection logic - now it gets triggered even when you navigate by keyboard

File: webapp/src/views/projects/translations/Filters/useFiltersContent.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    const newFilters = toggleFilter(filtersObj, availableFilters, rawValue);
+    dispatch({ type: 'SET_FILTERS', payload: newFilters });
+  };
+
+  const availableFilters = useAvailableFilters(selectedLanguages);
+
+  availableFilters.forEach((group, i1) => {
+    if (!group.options?.length) {
+      return;
+    } else {
```
Comment (JanCizmar) [inline webapp/src/views/projects/translations/Filters/useFiltersContent.tsx:31]: Useless else.

File: webapp/src/views/projects/translations/TranslationHeader/TranslationControlsCompact.tsx [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+                    size="small"
+                    className={classes.iconButton}
+                    onClick={() => setSearchOpen(true)}
+                  >
+                    <Search />
+                  </IconButton>
+                </div>
+              </Badge>
+
+              <Badge color="primary" badgeContent={activeFilters?.length}>
```
Comment (JanCizmar) [inline webapp/src/views/projects/translations/TranslationHeader/TranslationControlsCompact.tsx:217]: Maybe extract the menus to separate component?

File: webapp/src/views/projects/translations/Filters/useFiltersContent.tsx (hunk 1 above)
Comment (stepan662) [reply webapp/src/views/projects/translations/Filters/useFiltersContent.tsx]: 👍

File: webapp/src/views/projects/translations/TranslationHeader/TranslationControlsCompact.tsx (hunk 2 above)
Comment (stepan662) [reply webapp/src/views/projects/translations/TranslationHeader/TranslationControlsCompact.tsx:210]: Ok

**Review by JanCizmar — APPROVED** [review]

**Pull request merged**